jobbergate-core/jobbergate_core/version.py

```diff
@@ -0,0 +1,9 @@
+"""
+Provide the version of the package.
+"""
+try:
+    from importlib import metadata  # type: ignore
+except ImportError:
+    import importlib_metadata as metadata  # type: ignore # Python < 3.8
+
+__version__ = metadata.version(__package__ or __name__)
```

Comment:
I just see this variable is being used during tests. What's the purpose and utility of it?

Reply:
The version is available as metadata for end users that installed it from Pypi, for instance:

```python
>>> import jobbergate_core
>>> print(jobbergate_core.__version__)
3.4.3
```

It is a small touch, be can be useful sometimes.

Reply:
The reference came from here: https://stackoverflow.com/questions/67085041/how-to-specify-version-in-only-one-place-when-using-pyproject-toml